### PR TITLE
Jb54046

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last commit belongs to this PR

File: sdk-setup/src/sdk-manage [hunk 1]
```diff
@@ -1469,6 +1469,7 @@ synchronise_target() {
 # Qt5 qml imports are here
 + $libdir/qt5/qml**
 + /usr/include**
++ $libdir/*/include**
```
Comment (vigejolla) [inline sdk-setup/src/sdk-manage:1472]: What about the /usr/share dirs which were mentioned in the bug?

Comment (martyone) [reply sdk-setup/src/sdk-manage:1472]: Those from cmake are already covered, those from libtool are not relevant IIUC.

**Review by vigejolla — APPROVED** [review]
LGTM

File: sdk-setup/src/sdk-manage (hunk 1 above)
Comment (vigejolla) [reply sdk-setup/src/sdk-manage:1472]: Actually, cmake is *not* covered as far as I can tell: /usr/lib/cmake is synced, but /usr/share/cmake/Modules and /usr/share/cmake/Modules and /usr/share/cmake/include, which were listed in the bug, are not.

Comment (martyone) [reply sdk-setup/src/sdk-manage:1472]: Indeed :( and it's the commit on the very bottom... aargh

Comment (martyone) [reply sdk-setup/src/sdk-manage:1472]: I will push another commit directly to master after merging all the PRs.

Comment (martyone) [reply sdk-setup/src/sdk-manage:1472]: https://github.com/sailfishos/sdk-setup/commit/57ec7b54c62522b42b2257be16ad6f45d2f494ae

**Pull request merged**